subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/PluginAccessorsClassPath.kt

```diff
@@ -333,7 +333,7 @@ fun BufferedWriter.appendSourceCodeForPluginAccessors(
                     format(
                         """
                         /**
-                         * The `$id` plugin implemented by [$implementationClass].
+                         * The `$id` plugin implemented by [${implementationClass.replace('$', '.')}].
```

Comment:
:lipstick: I would move that computation outside of the string template.

`implementationClass` is a _binary name_ of a class. This is about converting from a _binary name_ to a _source name_ for that class.

We have an existing similar function at https://github.com/gradle/gradle/blob/f6eeed6aaf58643658b61593de8e4e95b17d2c59/subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/codegen/ApiTypeProvider.kt#L571

Reply:
Fixed. PTAL.